jcl/src/java.base/share/classes/java/security/AccessControlContext.java

```diff
@@ -521,7 +521,11 @@ static int checkPermWithCachedPDsImplied(Permission perm, Object[] toCheck, Acce
 					}
 				}
 			}
+			/*[IF Java11]*/
```

Comment:
Please use Java 9 instead of 11, no reason not to fix these releases as well. Although Java 9 is out of service its still somewhat alive for cmake and macOS atm. Also there will be one more build of Java 10 in the 0.11.0 release.

Reply:
Updated JPP flag to `Sidecar19-SE-OpenJ9`. Manually verified that `pConfig` still compiles with updated `ProtectionDomain` class.
Please have another look.